README.rst

```diff
@@ -100,7 +100,11 @@ Arguments to ``get_version()`` (see below) may be passed as a dictionary to
     from setuptools import setup
     setup(
         ...,
-        use_scm_version = {"root": "..", "relative_to": __file__},
+        use_scm_version = {
+            "root": "..",
+            "relative_to": __file__,
+            "local_scheme": "node-and-timestamp"
```

Comment:
```suggestion
            "local_scheme": "node-and-timestamp"  # the default
```

Reply:
Hmm, `config.py` says: `DEFAULT_LOCAL_SCHEME = "node-and-date"`...

Reply:
good catch, i think i clearly need more coffee